makefiles/coreutils.mk

```
@@ -32,7 +33,10 @@ coreutils: coreutils-setup gettext libgmp10 libxcrypt
 else # (,$(findstring darwin,$(MEMO_TARGET)))
 coreutils: coreutils-setup gettext libgmp10
 endif # (,$(findstring darwin,$(MEMO_TARGET)))
-	cd $(BUILD_WORK)/coreutils && ./configure -C \
+	cd $(BUILD_WORK)/coreutils && \
+			rm lib/config.hin && \
+			autoreconf -fi && \
+			./configure -C \
```

Do this in two lines rather than 3.

```suggestion
	rm -rf $(BUILD_WORK)/coreutils/lib/config.hin
	cd $(BUILD_WORK)/coreutils && autoreconf -fi && ./configure -C \
```